bci_tester/data.py

```diff
@@ -842,6 +842,23 @@ def create_BCI(
     forwarded_ports=[PortForwarding(container_port=80)],
 )
 
+_KUBECTL_VERSION_OS_MATRIX: Tuple[Tuple[str, Tuple[str, ...]], ...] = (
+    ("1.28", ("15.6", "tumbleweed")),
+    ("1.30", ("15.7")),
+    ("1.29", ("tumbleweed")),
+    ("1.30", ("tumbleweed")),
+    ("1.31", ("tumbleweed")),
```

Comment:
The second element is not a tuple, you're missing the trailing comma:
```
>>> type(("foo"))
<class 'str'>
>>> type(("foo",))
<class 'tuple'>
```

That's also why the whole CI is red.

```suggestion
    ("1.30", ("15.7",)),
    ("1.29", ("tumbleweed",)),
    ("1.30", ("tumbleweed",)),
    ("1.31", ("tumbleweed",)),
```

Reply:
fixed